webapp/src/components/modals/channel_subscriptions/select_channel_subscription.tsx

```diff
@@ -150,21 +221,39 @@ export default class SelectChannelSubscriptionInternal extends React.PureCompone
             titleMessage = <h2 className='text-center'>{'Jira Subscriptions'}</h2>;
         }
 
+        const subscriptionTemplateTitle = <h2 className='text-center'>{'Jira Subscription Templates'}</h2>;
         const showInstanceColumn = this.props.installedInstances.length > 1;
         let subscriptionRows;
+        let subscriptionTemplateRows;
+        const columns = (
+            <thead>
+                <tr>
+                    <th scope='col'>{'Name'}</th>
+                    <th
+                        className='th-col'
+                        scope='col'
+                    >{'Project'}</th>
+                    {showInstanceColumn &&
+                    <th
+                        className='th-col'
+                        scope='col'
+                    >{'Instance'}</th>
+                    }
+                    <th
+                        className='th-col'
+                        scope='col'
+                    >{'Actions'}</th>
+                </tr>
+            </thead>
+        );
```

Comment:
this is fine if Table layout was used before in this project, but we should prevent using Table in HTML now as they are not good and kind of deprecating now, it is advised to not use HTML table rather use `div` and CSS to create table like layout

Reply:
Yeah, actually the table layout is present on the same page for subscriptions so to make it consistent with this I have also used those.